### ISSUE TITLE
The same code, trained in the Ubuntu environment, Loss can converge, but under Windows training, Loss can not converge.

### ISSUE THREAD

**Issue by TCBocean** [tensorflow/tensorflow#33956]
I ran a project on github, the project address is https://github.com/qxde01/keras-alchemy, I tried to train a mobilenetv2 image classifier directly by running the `train_cifar100_classify.py` file, but I encountered some problems at runtime. .

My Windows environment:
```
Windows 10
Tensorflow-gpu 2.0
Keras 2.3.1
CUDA 10.0
```
My Ubuntu environment:
```
Ubuntu 16.04
Tensorflow-gpu 1.12
Keras 2.2.4
CUDA 9.0
```

I changed all `if tf.__version__<'2.0':` results to true at runtime to use keras as a front end to reduce version incompatibility issues.

The final result, as the title says, is trained on Windows and Loss is completely unable to converge. What is the reason for this? Can someone help me analyze it? thank you very much!
